src/pages/Favorites/Favorites.page.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useContext } from 'react';
+
+import VideoList from '../../templates/VideoList/VideoList';
+import { ThemeContext } from '../../providers/Theme/Theme.provider';
+import theme from '../../providers/Theme/Theme';
+
+import {
+  ContentMain,
+  TitleDiv,
+  ContDiv,
+  ContDivFst,
+  PrevBtn,
+  NextBtn,
+  NoVideos,
+  NoVideosDiv,
+} from './Favorites.page.styled';
+
+function favStorage() {
+  const values = [];
+  const keys = Object.keys(localStorage);
+
+  for (let i = 0; i < keys.length; i++) {
+    values.push(JSON.parse(localStorage.getItem(keys[i])));
+  }
+
+  return values;
+}
+
+function Favorites() {
+  const themeContext = useContext(ThemeContext);
+  const { mode } = themeContext.state;
+  const data = favStorage();
```

Comment:
This is also too generic, this can be named something like `favoritesList`, `favoritesData`, `favoritesItems`, etc